GoogleForCreators/web-stories-wp
# Shopping: Detect WooCommerce status

## Context



## Summary
Add a check to see if the woocommerce plugin is installed. If not installed when show an error / warning message to user in the shopping settings in the dashboard. We will also need to check to see if the plugin is activated ( and setup correctly ).

## Relevant Technical Choices



## To-do



## User-facing changes
<img width="1027" alt="Screenshot 2022-05-23 at 16 28 02" src="https://user-images.githubusercontent.com/237508/169854316-2bbd5216-1077-46ca-b38f-63aed5bc90ba.png">
<img width="1005" alt="Screenshot 2022-05-23 at 16 26 12" src="https://user-images.githubusercontent.com/237508/169854324-6e6aa454-515c-4901-9240-f583db3b2d81.png">
<img width="1022" alt="Screenshot 2022-05-23 at 16 25 52" src="https://user-images.githubusercontent.com/237508/169854328-977bc42b-539a-40ed-a1d0-c641f6273386.png">



## Testing Instructions






This PR can be tested by following these steps:

1. Enable shopping experiment. 
2. Go to dashboard > settings. 
3. Change shopping setting to woocommerce. 
4. See message saying to install woocommerce. 
5. Install woocommerce ( do not activate )
6. Go to dashboard > settings. 
7. See message saying to active woocommerce. 
8. Activate woocommerce. 
9. See manage message. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11383
